Puppet/Puppet.xcodeproj/project.pbxproj

```diff
@@ -1307,7 +1307,7 @@
 				DEVELOPMENT_TEAM = "";
 				GCC_OPTIMIZATION_LEVEL = 0;
 				INFOPLIST_FILE = NotifExtension/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
Notification extension is iOS 10+ functionality

Reply:
Damn! How did that happen.
Thx for the fix Ivan!